apps/sensenet/src/components/settings/stats-storage-widget.tsx

```diff
@@ -0,0 +1,148 @@
+import { useRepository } from '@sensenet/hooks-react'
+import { createStyles, makeStyles, Paper, useTheme } from '@material-ui/core'
+import React, { useEffect, useState } from 'react'
+import { useWidgetStyles } from '../../globalStyles'
+import { useLocalization } from '../../hooks'
+import { round } from '../dashboard'
+import { DashboardData } from '../dashboard/types'
+import { MultiPartProgressLine } from './multi-part-progress-line'
+
+const useStyles = makeStyles(() => {
+  return createStyles({
+    rowContainer: {
+      display: 'flex',
+      justifyContent: 'space-between',
+      padding: '10px 0',
+    },
+  })
+})
+
+export const StorageWidget: React.FunctionComponent = () => {
+  const classes = useStyles()
+  const widgetClasses = useWidgetStyles()
+  const theme = useTheme()
+  const localization = useLocalization().settings
+  const numberFormatter = new Intl.NumberFormat('en-US')
+  const repository = useRepository()
+  const [data, setData] = useState<DashboardData>()
+
+  useEffect(() => {
+    ;(async () => {
+      const response = await repository.executeAction<any, DashboardData>({
+        idOrPath: '/Root',
+        name: 'GetDashboardData',
+        method: 'GET',
+        oDataOptions: {
+          select: ['Plan'],
+        },
+      })
+
+      setData(response)
+    })()
+  }, [repository])
+
+  if (!data) return null
+
+  const allUsage =
+    data.usage.storage.files +
+    data.usage.storage.content +
+    data.usage.storage.oldVersions +
+    data.usage.storage.log +
+    data.usage.storage.system
+
+  return (
+    <div className={widgetClasses.root}>
+      <Paper elevation={0} className={widgetClasses.container}>
+        <div className={classes.rowContainer}>
+          <span>{localization.storage}</span>
+          <span>
+            {localization.used(
+              `${numberFormatter.format(round(allUsage / 1024 / 1024))} MB`,
+              `${numberFormatter.format(round(data.subscription.plan.limitations.storage / 1024))} GB`,
```

Comment:
is it good to compare values with different units?

Reply:
Check the design: https://miro.com/app/board/o9J_klNk0R4=/